airflow/ui/src/pages/DagsList/DagsList.tsx

```diff
@@ -200,21 +200,31 @@ export const DagsList = () => {
           <Heading py={3} size="md">
             {pluralize("DAG", data?.total_entries)}
           </Heading>
-          {display === "card" ? (
-            <Select
-              data-testid="sort-by-select"
-              onChange={handleSortChange}
-              placeholder="Sort by…"
-              value={orderBy}
-              variant="flushed"
-              width="200px"
-            >
-              <option value="dag_id">Sort by DAG ID (A-Z)</option>
-              <option value="-dag_id">Sort by DAG ID (Z-A)</option>
-            </Select>
-          ) : (
-            false
-          )}
+          <Select
+            data-testid="sort-by-select"
+            onChange={handleSortChange}
+            placeholder="Sort by…"
+            value={orderBy}
+            variant="flushed"
+            width="200px"
+          >
+            <option value="dag_display_name">Sort by Display Name (A-Z)</option>
+            <option value="-dag_display_name">
+              Sort by Display Name (A-Z)
+            </option>
+            <option value="next_dagrun">Sort by Next DAG Run (A-Z)</option>
+            <option value="-next_dagrun">Sort by Next DAG Run (Z-A)</option>
+            <option value="last_run_state">Sort by Last Run State (A-Z)</option>
+            <option value="-last_run_state">
+              Sort by Last Run State (Z-A)
+            </option>
+            <option value="last_run_start_date">
+              Sort by Last Run Start Date (A-Z)
+            </option>
+            <option value="-last_run_start_date">
+              Sort by Last Run Start Date (Z-A)
```

Comment:
A lot of these aren't actually alphabetical searches. Two options:

1. Just put a +/- indicator, even better if we used the same up and down arrow icons as the table sort function
2. Only have one option per select and each time you click it it cycles through asc, desc or removing that sort. With the symbols to match

Reply:
Let's just change the start_date from (A-Z) to (latest-earliest) and we can do the rest later.

Reply:
The problem with 2 option is that it would not be user friendly in case of multi selection.

Reply:
Yeah, we can optimize later.